### ISSUE TITLE
https://www.w3.org/pubrules redirects to https://www3.org/pubrules which is a 404

### ISSUE THREAD

**Issue by TallTed** [w3c/specberus#1709]
TL;DR: redirect from `https://www.w3.org/pubrules`
                  to `https://www3.org/pubrules/`
        should be to `https://w3.org/pubrules/`

The longer version --

https://www.w3.org/2004/02/02-pubrules

-- has a link to the "latest version" --

http://www.w3.org/Guide/pubrules

-- which has some redirects, which wind up at a bad URL --
```
$ curl -kIL http://www.w3.org/Guide/pubrules
HTTP/1.1 301 Moved Permanently
Date: Wed, 22 Feb 2023 18:33:07 GMT
Connection: keep-alive
Cache-Control: max-age=3600
Expires: Wed, 22 Feb 2023 19:33:07 GMT
Location: https://www.w3.org/Guide/pubrules
Set-Cookie: __cf_bm=u.w0ZhCxpgkJx5fATNOEI9OVjvAxO87O1nrRWmZLOwU-1677090787-0-AfNT5Evu/kU9unX7SARAs8IRYvCngnZvC5PUvDlmTt/RRnvH0UQ5LBafcjDhfalmYm5CtBsxJpFD9CpO5dwSMFg=; path=/; expires=Wed, 22-Feb-23 19:03:07 GMT; domain=.w3.org; HttpOnly; SameSite=None
Server: cloudflare
CF-RAY: 79d9ceeb4f73c413-EWR
alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400

HTTP/2 302 
date: Wed, 22 Feb 2023 18:33:07 GMT
content-type: text/html; charset=iso-8859-1
content-length: 211
location: https://www.w3.org/pubrules
cache-control: max-age=21600
expires: Thu, 23 Feb 2023 00:33:07 GMT
x-backend: www-mirrors
x-request-id: 79d9ceec391819d3
strict-transport-security: max-age=15552000; includeSubdomains; preload
content-security-policy: upgrade-insecure-requests
cf-cache-status: BYPASS
set-cookie: __cf_bm=i75MbTXFdbbXYhMahZ015fm7VMIiClPeUvP25ASp5Kg-1677090787-0-ATRG0AzIZVCMKQ7TWQ++dXTSmS4XGFuXfnHB7zeAe6RDfZdSWc+g5zjOgA0Eft24R4TnVjgGDDYvVZHFDUwfiwM=; path=/; expires=Wed, 22-Feb-23 19:03:07 GMT; domain=.w3.org; HttpOnly; Secure; SameSite=None
server: cloudflare
cf-ray: 79d9ceec391819d3-EWR
alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400

HTTP/2 302 
date: Wed, 22 Feb 2023 18:33:07 GMT
content-length: 0
location: https://www3.org/pubrules/
cache-control: no-cache
cf-cache-status: MISS
set-cookie: __cf_bm=x24Iwa5JQmFqaDR1mv4Gz1e1OzRPswiuFh8mHAxs.sI-1677090787-0-ASy45OtZmnerCOcfxMcu4y7ofB/WPDHoOiBndKv+0VTSHxzHyIpFteNTc8QOXH2KPfAaBYHZ6S6apUTDcs29pnc=; path=/; expires=Wed, 22-Feb-23 19:03:07 GMT; domain=.w3.org; HttpOnly; Secure; SameSite=None
server: cloudflare
cf-ray: 79d9ceecea4919d3-EWR
alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400

HTTP/1.1 404 Not Found
Date: Wed, 22 Feb 2023 18:33:07 GMT
Content-Type: text/html
Content-Length: 867
Connection: keep-alive
Server: Apache/2
Last-Modified: Fri, 10 Jan 2020 16:05:10 GMT
Accept-Ranges: bytes
Age: 0
```

The Link Checker has MANY words for you, too!

   https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2F2004%2F02%2F02-pubrules&hide_type=all&depth=&check=Check

I don't know where exactly this needs to be fixed, but it does.

/cc @afs @gkellogg @pchampin @rdfguy

**Comment by deniak** [comment]
@TallTed Thank you for your report. We did have a misconfigured redirect. This should now be fixed.
